java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -247,4 +251,24 @@ public void set_missing_value_and_returnOldValue_is_null(BaseClient client) {
         String data = client.set("another", ANOTHER_VALUE, options).get();
         assertNull(data);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void config_rewrite(BaseClient client) {
+        var exception = assertThrows(ExecutionException.class, () -> client.configRewrite().get());
+        assertAll(
+                () -> assertTrue(exception.getCause() instanceof RequestException),
+                () ->
+                        assertTrue(
```

Comment:
Shoham asked me to remove the error message checks in my PR, although they check the error message in the other clients
https://github.com/aws/glide-for-redis/pull/974#discussion_r1491537664

Reply:
Since IT runs cluster without a config file, this command always fails in IT.
Thus, this test is the only test which I can do for that command.

Reply:
Docs https://redis.io/commands/config-rewrite/ say
> However if the server was started without a configuration file at all, the CONFIG REWRITE will just return an error.

Reply:
Remove it. 
We're running against multiple version so Redis, and we don't want to check for version-specific error messages when they update Redis.

Reply:
Removed in 29cdfa49

Reply:
https://github.com/aws/glide-for-redis/blob/main/python/python/tests/test_async_client.py#L831 all their tests check for error messages. This includes Node and Py maybe she is mistaken?

Reply:
IMO we should keep the test. In case if CI get a newer redis version with another error message, we will fix the test.
Meanwhile I do `contains` check, not `equals`